docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -227,7 +227,43 @@ Note also that the main navigation logo uploaded via the admin panel supersedes
 
 #### Favicon
 
-To update the favicon, put a favicon file in the `./src/admin/extensions` folder and update the `config.head.favicon` key in the [admin panel configuration](#configuration-options).
+Replacing the Strapi favicon with a custom favicon.
+:::tip
+This same process may be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [#logos](#)
```

Comment:
This link's target is empty. Which section of the documentation would you like to link to?

Reply:
Hi, the link works as intended. It brings you to the #configuration-options.

Please double check this; I am using Firefox latest on Ubuntu 22.04. 

https://documentation-bsk2xvgex-strapijs.vercel.app/developer-docs/latest/development/admin-customization.html#customization-options

That link brings you to the same place the [#logos](#customization-options). Only difference is, you don't need the absolute URL because you are already on that page.

Reply:
ohh i see, that is a typo on another line that is similar. I fixed it. Good catch.